src/EasyCaching.HybridCache/HybridCachingProvider.cs

```diff
@@ -81,39 +74,15 @@ string name
             this._logger = loggerFactory?.CreateLogger<HybridCachingProvider>();
 
             // Here use the order to distinguish traditional provider
-            var local = factory.GetCachingProvider(_options.LocalCacheProviderName);
-            if (local.IsDistributedCache) throw new NotFoundCachingProviderException("Can not found any local caching providers.");
-            else this._localCache = local;
+            this._localCache = factory.GetCachingProvider(_options.LocalCacheProviderName);
 
             // Here use the order to distinguish traditional provider
-            var distributed = factory.GetCachingProvider(_options.DistributedCacheProviderName);
-
-            if (!distributed.IsDistributedCache) throw new NotFoundCachingProviderException("Can not found any distributed caching providers.");
```

Comment:
When distibuted cache is decorated with circuit breaker only (it's the best option when using it with HybridCache) and fails on initialization, we fail on accessing IsDistributedCache.
It's hard to pass actual IsDistributedCache to decorator all the way from cache initialization, so I think it's would be better to live without this check.